### PR TITLE
[FEAT]:cards_section : Added the cards

### PR DESCRIPTION
**Pull request by shlok-2003**
Added Card Section resolving issue no #7 

DIsclamer: The background image has not been added, since the component was divided for two issues

### CODE REVIEW

File: src/components/CardSection/index.astro [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    {
+        "cardImg": "Chart.svg",
+        "cardAlt": "Chart Icon - Representing Effective Resume Scanner",
+        "cardTitle": "Effective Resume Scanner",
+        "cardText": "You get several free uploads to our resume scanner. If you have different versions of your resumes (and you should, especially if you are applying to different industries!), you can test"
+    },
+]
+---
+
+<section class="px-16 my-5 grid grid-cols-1 sm:grid-cols-4 lg:grid-cols-3 gap-16">
```
Comment (JayeshVP24) [inline src/components/CardSection/index.astro:24]: change gap
make gap smaller for mobile and larger for desktop

Comment (JayeshVP24) [inline src/components/CardSection/index.astro:24]: make padding px-4 for mobile px-16 for larger screens

File: src/components/CardSection/index.astro [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            <div class="pt-7 pb-4 px-5 flex flex-col col-span-1 sm:max-lg:col-span-2 sm:max-lg:last:col-start-2 bg-[#24294D] rounded-2xl">
+                <img 
+                    src={`/CardSection/${item.cardImg}`} alt={item.cardAlt}
+                    class="ml-3 mb-5 w-max object-center h-16 sm:h-20 md:h-24 object-cover select-none"
+                />
+
+                <div class="flex flex-col justify-between h-[calc(100%-25%)]">
+                    <h1 class="inline-block font-rubik text-white font-bold text-2xl sm:text-3xl">{item.cardTitle}</h1>
+                    
+                    <p class="font-sans font-normal text-gray-300 text-sm sm:text-base">{item.cardText}</p>
```
Comment (JayeshVP24) [inline src/components/CardSection/index.astro:35]: add little padding top

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]

File: src/components/CardSection/index.astro [hunk 3]
```diff
@@ -21,15 +21,15 @@ const card = [
...
                 <img 
                     src={`/CardSection/${item.cardImg}`} alt={item.cardAlt}
                     class="ml-3 mb-5 w-max object-center h-16 sm:h-20 md:h-24 object-cover select-none"
                 />
 
-                <div class="flex flex-col justify-between h-[calc(100%-25%)]">
+                <div class="flex flex-col justify-between pt-2 h-[calc(100%-25%)]">
                     <h1 class="inline-block font-rubik text-white font-bold text-2xl sm:text-3xl">{item.cardTitle}</h1>
                     
                     <p class="font-sans font-normal text-gray-300 text-sm sm:text-base">{item.cardText}</p>
```
Comment (JayeshVP24) [inline src/components/CardSection/index.astro:35]: add little padding above item.cardText not above item.cardTitle

**Comment by shlok-2003** [comment]
Made the corresponding changes in the file,

File: src/components/CardSection/index.astro [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    {
+        "cardImg": "Chart.svg",
+        "cardAlt": "Chart Icon - Representing Effective Resume Scanner",
+        "cardTitle": "Effective Resume Scanner",
+        "cardText": "You get several free uploads to our resume scanner. If you have different versions of your resumes (and you should, especially if you are applying to different industries!), you can test"
+    },
+]
+---
+
+<section class="my-5 grid grid-cols-1 sm:grid-cols-4 lg:grid-cols-3 px-4 sm:px-8 md:px-12 lg:px-16 gap-8 sm:gap-10 md:gap-14 lg:gap-16">
```
Comment (JayeshVP24) [inline src/components/CardSection/index.astro:24]: remove grid, change to `flex flex-wrap`

**Review by JayeshVP24 — APPROVED** [review]
LGTM

**Pull request merged**